ProjFS.Mac/PrjFSKext/KauthHandler.cpp

```diff
@@ -46,7 +52,7 @@ static bool TryReadVNodeFileFlags(vnode_t vn, vfs_context_t _Nonnull context, ui
 KEXT_STATIC_INLINE bool FileFlagsBitIsSet(uint32_t fileFlags, uint32_t bit);
 static inline bool TryGetFileIsFlaggedAsInRoot(vnode_t vnode, vfs_context_t _Nonnull context, bool* flaggedInRoot);
 KEXT_STATIC_INLINE bool ActionBitIsSet(kauth_action_t action, kauth_action_t mask);
-
+static bool CurrentProcessIsOwnedOrWasSpawnedByRegularUser();
```

Comment:
Naming nit: Is there a difference between owned and spawned as far as we're concerned or can we rename this to simply be Owned?

Reply:
"Owned" to me implies "UID is equal to", which is definitely not what this function checks. "Spawned" covers the ownership bit too though so we could reduce it down to `CurrentProcessWasSpawnedByRegularUser()` to make it less wordy.

Reply:
> we could reduce it down to CurrentProcessWasSpawnedByRegularUser() to make it less wordy.

+1 for switching to `CurrentProcessWasSpawnedByRegularUser`